images/installer-origin-release/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# This Dockerfile is used to create the image quay.io/coreos/origin-release:bazel used in CI
+# to build smoke tests.
+# EX: docker build --rm -t this-image:latest .
+
+FROM docker.io/openshift/origin-release:golang-1.10
+
+RUN wget https://copr.fedorainfracloud.org/coprs/vbatts/bazel/repo/epel-7/vbatts-bazel-epel-7.repo && \
+    mv vbatts-bazel-epel-7.repo /etc/yum.repos.d/
```

Comment:
You can [still][1] drop this `mv`.

[1]: https://github.com/openshift/installer/pull/223/#discussion_r216013284